packages/react-native/sdks/hermes-engine/utils/build-apple-framework.sh

```diff
@@ -31,20 +31,29 @@ function use_env_var_or_ruby_prop {
   fi
 }
 
+function use_env_var {
+  if [[ -n "$1" ]]; then
+    echo "$1"
+  else
+    echo "error: Missing DEPLOYMENT_TARGET environment variable"
```

Comment:
can we pass the name of the DEPLOYMENT_TARGET variable?

```suggestion
    echo "error: Missing $2 environment variable"
```

and we call it like:
```sh
function get_ios_deployment_target {
use_env_var "${IOS_DEPLOYMENT_TARGET}" "IOS_DEPLOYMENT_TARGET"
}
```
So we have a better error message.

Reply:
Fixed